istio-client/src/main/java/me/snowdrop/istio/client/IstioClient.java

```diff
@@ -75,6 +75,10 @@ public IstioResource registerCustomResource(final IstioResource resource) {
         return client.createCustomResources(resource).get(0);
     }
 
+    public Adapter getClient() {
```

Comment:
Do you need access to the Adapter?

Reply:
Ideally yes, so I only need to store the `istio-client`.

Reply:
Look at https://github.com/snowdrop/istio-java-api/blob/master/istio-client/src/main/java/me/snowdrop/istio/client/IstioClient.java#L93, would that work for you?

Reply:
yes it is perfect. You can close this PR then :)

Reply:
Sorry for the extra work, I should have pushed earlier…

Reply:
No worries it is just to getters :)